wizard-frontend/src/App.tsx

```diff
@@ -0,0 +1,56 @@
+import { Toaster } from "@/components/ui/toaster";
+import { Toaster as Sonner } from "@/components/ui/sonner";
+import { TooltipProvider } from "@/components/ui/tooltip";
+import {
+  QueryClient,
+  QueryClientProvider,
+  useQuery,
+} from "@tanstack/react-query";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Homepage from "./pages/Homepage";
+import NotFound from "./pages/NotFound";
+import { UserDataContext } from "./components/UserDataContext";
+
+const queryClient = new QueryClient();
+
+const InnerApp = () => {
+  const { error, isLoading, data } = useQuery({
+    queryKey: ["userIsReturning"],
+    queryFn: () =>
+      fetch(window.location.href + "api/v1/is-returning").then(async (res) =>
+        res.ok && (await res.text()) === "true" ? true : false
+      ),
+  });
+
+  if (error) {
+    console.log(error)
```

Comment:
No notification anywhere that we have an error? I think we should have something, maybe there's something wrong with the user setup, and this might leave them confused until they randomly check the browser console.

Reply:
Also, this won't just display `[Object object]`, right?

And lastly, `console.error(error)` (I think that's the correct one?).